assets/extensions/examples/online-extensions.js

```diff
@@ -58,27 +57,27 @@ angular
 
           system_status_elem_mobile.append(system_status_elem.clone());
           system_status_elem_mobile.css('display', '');
```

Comment:
If we're only adding the element when there's content, can we remove the `display: none` above?